app/models/send_message.rb

```diff
@@ -0,0 +1,15 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness
+require 'twilio-ruby'
+class SendMessage
+
+  def self.text
+    account_sid = 'AC2eb5874bc5580bf392654c9381f1b7ae'
+    auth_token  =  "2ad9acc648ef6b4f11e889709b928d18"
+
+    @client = Twilio::REST::Client.new account_sid, auth_token
+
+    @client.account.messages.create({
+    	:from => '+17604528410', :to => '+18587227866', :body => "your order was succefully placed"
```

Comment:
Line is too long. [96/80]<br>Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.